ParkingLot/src/main/java/Application.java

```diff
@@ -1,61 +0,0 @@
-import dao.TicketsDao;
-import factories.CostComputationFactory;
-import factories.ParkingSpotManagerFactory;
-import models.ParkingSpots.FourWheelerParkingSpot;
-import models.ParkingSpots.ParkingSpot;
-import models.ParkingSpots.TwoWheelerParkingSpot;
-import models.Ticket;
-import models.Vehicle;
-import models.VehicleType;
-import services.EntranceGate;
-import services.ExitGate;
-
-
-import java.util.ArrayList;
-import java.util.List;
-
-public class Application {
-
-    public static void main(String[] args) throws Exception {
-        TicketsDao ticketsDao = new TicketsDao();
-        CostComputationFactory costComputationFactory = new CostComputationFactory();
-        List<ParkingSpot> twoWheelerParkingSpotList = new ArrayList<>();
-        List<ParkingSpot> fourWheelerParkingSpotList = new ArrayList<>();
-        List<Ticket> ticketList = new ArrayList<>();
-        for(int i = 0; i< 5; i++){
-            twoWheelerParkingSpotList.add(new TwoWheelerParkingSpot());
-        }
-        for(int i = 0; i< 3; i++){
-            fourWheelerParkingSpotList.add(new FourWheelerParkingSpot());
-        }
-
-        ParkingSpotManagerFactory parkingSpotManagerFactory = new ParkingSpotManagerFactory(twoWheelerParkingSpotList, fourWheelerParkingSpotList);
-
-        EntranceGate primaryEntryGate = new EntranceGate(parkingSpotManagerFactory, ticketsDao);
```

Comment:
Can you elaborate on what is the expectation from such a service? 

We are using a spot manager to manage all the parking spots, this can loosely translate to parking lot service.